src/Backend/Modules/Location/Js/Location.js

```diff
@@ -70,7 +69,7 @@ export class Location {
       // add event listener
       google.maps.event.addListener(marker, 'dragend', function () {
```

Comment:
do you need the `.bind` on line 72 when you use an arrow function?

Reply:
@Katrienvh fixed in fec96fedb131dbc00520ea861339093e2eefe5c4